### PR TITLE
Refine TF Object Detection Examples For New API

### PR DESCRIPTION
**Pull request by zehao-intel**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not: No

## Description

detail description 
JIRA ticket: 2594

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR : as title

## How has this PR been tested?

how to reproduce the test (including hardware information) : Extension

## Dependency Change?

any library dependency introduced or removed: No

### CODE REVIEW

**Comment by lvliang-intel** [comment]
Please update the examples/README.md for the refined examples.

File: examples/tensorflow/object_detection/yolo_v3/quantization/ptq/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+This script will download the *train*, *validation* and *test* COCO datasets. Furthermore it will convert them to
+tensorflow records using the `https://github.com/tensorflow/models.git` dedicated script.
+
+#### Manual dataset download
+Download CoCo Dataset from [Official Website](https://cocodataset.org/#download).
+
+## Get Quantized Yolo-v3 model with Neural Compressor
+
+### 1.Config the yolo_v3.yaml with the valid cocoraw data path or the yolo_v3_itex.yaml if using the Intel Extension for Tensorflow.
```
Comment (chensuyue) [inline examples/tensorflow/object_detection/yolo_v3/quantization/ptq/README.md:85]: Please remove the yaml related description.

File: examples/tensorflow/object_detection/tensorflow_models/README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
+Step-by-Step
```
Comment (chensuyue) [inline examples/tensorflow/object_detection/tensorflow_models/README.md:1]: Should we remove this readme, most of the context duplicate with single model readme.

**Comment by chensuyue** [comment]
Please follow the latest example readme template to refine the readme structure.

**Comment by zehao-intel** [comment]
Need extension test for these models @chensuyue

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4107/artifact/report.html)

**Comment by zehao-intel** [comment]
Extension for yolo_v3: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4137/

**Comment by zehao-intel** [comment]
Rebuild extension of all other models except ckpt: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4153/

**Comment by zehao-intel** [comment]
Extension test: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4173/

**Comment by chensuyue** [comment]
> Extension test: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4173/

1. Please check acc and perf gap. 
2. Check CKPT model fp32 benchmark.

**Comment by zehao-intel** [comment]
Extension for models with ckpt: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4190/

**Comment by zehao-intel** [comment]
Extension for models with ckpt: https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-validation-top-mr-extension/4199/

File: examples/README.md [hunk 3]
```diff
@@ -246,49 +246,49 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>SSD ResNet50 V1</td>
     <td>Object Detection</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">ckpt</a></td>
+    <td><a href="https://github.com/intel/neural-compressor/tree/master/examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/master/examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">ckpt</a></td>
```
Comment (lvliang-intel) [inline examples/README.md:249]: just need to provide the relative path. 
./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq

File: examples/tensorflow/object_detection/tensorflow_models/faster_rcnn_inception_resnet_v2/quantization/ptq/README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+This document is used to list steps of reproducing TensorFlow Object Detection models tuning results. This example can run on Intel CPUs and GPUs.
+
+## Prerequisite
+
+
+### 1. Environment
+Recommend python 3.6 or higher version.
+
+#### 1. Install Intel® Neural Compressor
```
Comment (lvliang-intel) [inline examples/tensorflow/object_detection/tensorflow_models/faster_rcnn_inception_resnet_v2/quantization/ptq/README.md:12]: Remove the number at this level.

**Review by lvliang-intel — APPROVED** [review]

File: examples/README.md [hunk 5]
```diff
@@ -246,49 +246,49 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>SSD ResNet50 V1</td>
     <td>Object Detection</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">ckpt</a></td>
+    <td><a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">pb</a> / <a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">ckpt</a></td>
```
Comment (chensuyue) [inline examples/README.md:249]: Please fix the hyperlink issue.

Comment (chensuyue) [reply examples/README.md:249]: And also remove the `*` before new API examples. 
image

**Pull request merged**